Network/Socket/Constant.hsc

```diff
@@ -0,0 +1,47 @@
+module Network.Socket.Constant where
+
+#include "HsNet.h"
+
+import Data.Word (Word32)
+
+import Network.Socket.Types
+
+-- ---------------------------------------------------------------------------
+-- Utility Functions
+
+aNY_PORT :: PortNumber
+aNY_PORT = 0
+
+-- | The IPv4 wild card address.
+
+iNADDR_ANY :: HostAddress
+iNADDR_ANY = htonl (#const INADDR_ANY)
+
+-- | Converts the from host byte order to network byte order.
+foreign import CALLCONV unsafe "htonl" htonl :: Word32 -> Word32
+-- | Converts the from network byte order to host byte order.
+foreign import CALLCONV unsafe "ntohl" ntohl :: Word32 -> Word32
```

Comment:
Should `htonl` and `ntohl` live here? They aren't really constants, but endian utilities.

Reply:
Oh. We have the same defs in `Types`, too!